jobbergate-api/jobbergate_api/apps/garbage_collector.py

```diff
@@ -31,12 +34,9 @@ async def get_files_to_delete(session, table, bucket) -> set[str]:
 async def delete_files_from_bucket(bucket, files_to_delete: set[str]) -> None:
     """Delete files from the bucket."""
     for file_to_delete in files_to_delete:
-        try:
-            file = await bucket.Object(file_to_delete)
-            await file.delete()
-            logger.debug(f"Deleted file {file_to_delete} from bucket {bucket.name}")
-        except Exception as e:
-            logger.error(f"Failed to delete file {file_to_delete} from bucket {bucket.name}: {e}")
+        file = await bucket.Object(file_to_delete)
+        await file.delete()
```

Comment:
We could batch-delete them to avoid requests in a loop. I created a placeholder for it: https://app.clickup.com/t/18022949/PENG-1545

Reply:
Also a good tech debt item.